resources/views/site/blocks/feature_block.blade.php

```diff
@@ -24,13 +24,9 @@
             if ($block->input('override_event')) {
                 $items = $block->getRelated('events');
 
-                foreach ($items as $key => $event) {
-                    $eventWithMetas = $event->is_future;
-
-                    if (!$eventWithMetas) {
-                        unset($items[$key]);
-                    }
-                }
+                $items = $items->filter(function ($event) {
+                    return $event->is_future;
+                })->values()->take(4);
```

Comment:
👍

Reply:
I just learned how to use this and it's much cleaner

Reply:
python ruined how I handle objects lol